lib/puppet/functions.rb

```diff
@@ -108,6 +116,22 @@
 #     end
 #   end
 #
+# There is no requirement for direct mapping between parameter declarations and the parameters in the receiver so
+# the following example is also legal. Here the dispatch will ensure that *values in the receiver will
```

Comment:
here in github comments? It is the same as in markdown - backtick before and after (klick on edit to see how it is done). You can create a block by using three backticks before and after like this:
```
this is code
```